src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ResidentJudgeFtpaDecisionHandler.java

```diff
@@ -95,6 +97,15 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
 
         final String ftpaApplicantUpperCase = ftpaApplicantType.toUpperCase();
 
+        String ftpaDecisionOutcomeType = asylumCase.read(
+                        valueOf(String.format("FTPA_%s_RJ_DECISION_OUTCOME_TYPE", ftpaApplicantUpperCase)), String.class)
+                .orElseThrow(() -> new IllegalStateException("ftpaDecisionOutcomeType is not present"));
+
+        boolean isDlrmSetAside
+                = featureToggler.getValue(DLRM_SETASIDE_FEATURE_FLAG, false);
+
+        if (!ftpaDecisionOutcomeType.equals("reheardRule35")) {
```

Comment:
What happens if dlrm flag is not toggled on yet. reheardRule35 is still using the same fields as "Granted"?

Reply:
@ThomasKKC not exactly, rule 35 implements its own new fields, and DOESN'T use necessarily the fields of the other outcome types